src/main/java/com/openlattice/shuttle/dates/TimeZones.java

```diff
@@ -30,8 +39,34 @@
     public static final TimeZone America_LosAngeles = TimeZone.getTimeZone( "America/Los_Angeles" );
     public static final TimeZone America_Chicago    = TimeZone.getTimeZone( "America/Chicago" );
     public static final TimeZone America_Denver     = TimeZone.getTimeZone( "America/Denver" );
+    public static final TimeZone UTC                = TimeZone.getTimeZone( "UTC" );
 
     private TimeZones() {
     }
 
+    private static final Logger logger = LoggerFactory.getLogger( TimeZones.class );
+
+    public static Optional<TimeZone> checkTimezone( Optional<String> timezoneString ) {
+        if ( timezoneString.isPresent() ) {
+            String timezoneId = timezoneString.get();
+
+            Optional<TimeZone> timezone = Optional.of(TimeZone.getTimeZone( timezoneId ));
+
+            if ( timezone.orElse( Constants.DEFAULT_TIMEZONE ).getID().equals( timezoneId ) ) {
```

Comment:
I think, get() would be enough here instead of orElse, since TimeZone.getTimeZone will always return a timezone:
```
Returns:
the specified TimeZone, or the GMT zone if the given ID cannot be understood.
```